src/main/java/calculator/model/Expression.java

```diff
@@ -15,13 +15,14 @@
 
 	public static final String CALCULATE_SEPARATOR_COMMA = ",";
 	public static final String CALCULATE_SEPARATOR_COLON = ":";
+	public static final String SEPARATOR_ADD_PATTERN = "//(.)\n(.*)";
+	public static final Pattern PATTERN_COMPILE = Pattern.compile(SEPARATOR_ADD_PATTERN);
```

Comment:
```suggestion
	public static final Pattern PATTERN_COMPILE = Pattern.compile("//(.)\n(.*)");
```

상수를 사용하는데 장점중에 이름을 통해 의미를 빠르게 파악 할 수 있는 장점이 있습니다.
좀 더 직관적인 이름을 고민해보면 어떨까요 ? 🤔